### ISSUE TITLE
Dragging an image to anywhere in the PDP when the settings drawer is open, uploads the image for the first variant

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2478]
### Expected behavior
If you drag an image for the main image area, it should become the "main" product image

### Actual Behavior
It gets uploaded as an image for the first option (or probably whatever option is active for editing)

### Steps to Reproduce the Behavior
1. Login as an admin
1. Click on the edit icon for the variant so that the edit drawer opens
1. Drag an image into the "main" image area (over the + sign)
1. Observe that image is now uploaded for the first variant
1. Publish
1. Observe that it does not show up in the product grid because it is not the "main" image.

Also clicking on the plus sign and uploading that has the same behavior. So it requires a lot of extra effort to perform what is the most common use case.

### Versions
```
Node: 7.10.0
NPM: 5.0.3
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.10.0
Reaction: 1.3.0
Reaction branch: development
Docker: 17.03.1-ce
```

**Comment by zenweasel** [comment]
@spencern also agrees this is broken

**Comment by zenweasel** [comment]
Closed via #2506
